src/providers/proxy/proxy_ipnetworks.c

```diff
@@ -233,7 +233,7 @@ get_net_byname(struct proxy_resolver_ctx *ctx,
     if (ret == ENOENT) {
         /* Not found, make sure we remove it from the cache */
         DEBUG(SSSDBG_TRACE_INTERNAL, "Network [%s] not found, removing from "
-              "cache\n", name);
+              "cache\n", search_name);
```

Comment:
What about `DEBUG(SSSDBG_TRACE_INTERNAL, "Network [%s] found, saving into "` below?
Probably makes sense to print both `name` and `search_name`?

Reply:
`name` is initialized to `NULL` but no useful value is assigned at this point. That is the warning I'm fixing.

Reply:
But this is inconsistent to print `search_name` in one case and `name` in another case.

Reply:
I suggest to modify "found" message to something like "Network [%s -> %s] found..."